src/components/Article/Article.jsx

```diff
@@ -1,6 +1,68 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Article.css';
 
-// Create an Article function returning the HTML of article block
+function Article() {
```

Comment:
we usually make it arrow function
```suggestion
const Article = () => {
```